### PR TITLE
docs: watchman installation step removed

### PR DESCRIPTION
**Pull request by iamdevvalecha**
Fixes #

#### Changes proposed in this pull request:

-
-
-

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/installation/local.md [hunk 1]
```diff
@@ -6,9 +6,7 @@ You will need the following things properly installed on your computer.
 * **[Git](https://git-scm.com/)**
 * **[Node.js 14.x LTS](https://nodejs.org/)**
 * **[Yarn](https://yarnpkg.com/en/docs/install)**
-
-It is also recommended to have [watchman](https://facebook.github.io/watchman/docs/install) installed to speed up the `file-watcher/auto-build` service of the ember build server.
-
+*
```
Comment (iamareebjamal) [inline docs/installation/local.md:9]: Why this bullet point

**Comment by iamdevvalecha** [comment]
bullet point removed.

**Comment by iamdevvalecha** [comment]
@iamareebjamal please, check i have removed the bullet point sir.

**Comment by iamdevvalecha** [comment]
@iamareebjamal please reply sir.

**Review by maze-runnar — APPROVED** [review]

**Comment by iamdevvalecha** [comment]
@maze-runnar thanks bhaiya!

**Comment by iamdevvalecha** [comment]
@iamareebjamal Thanks.

**Pull request merged**